crypto/evp/evp_enc.c

```diff
@@ -911,13 +914,15 @@ int EVP_DecryptFinal_ex(EVP_CIPHER_CTX *ctx, unsigned char *out, int *outl)
     ret = ctx->cipher->cfinal(ctx->algctx, out, &soutl,
                               blocksize == 1 ? 0 : blocksize);
 
-    if (ret) {
-        if (soutl > INT_MAX) {
-            ERR_raise(ERR_LIB_EVP, EVP_R_FINAL_ERROR);
```

Comment:
this error has nothing to do with the padding, right?
it would be good to keep the ERR_raise when in this happens,
just translate the `if (ret) if (sout1 > INT_MAX)` to constant time.

Reply:
It actually has to do with the padding, the value of `ret` depends on the return value of `ossl_cipher_unpadblock`.

Reply:
yes, of course, but if I see it right, only the condition `ret != 0 && sout1 > INT_MAX`
has to be evaluated in constant time, but in that unlikely event it's okay to execute
a branch statement and call the ERR_raise.